psoc/grobot_workspace/grobot_module_controller.cydsn/dht11.c

```diff
@@ -0,0 +1,234 @@
+//#include <project.h>
+
+#include "dht11.h"
+#include <stdint.h>
+#include <dht.h>
+
+#define null 0
+
+void delay(void);
+void unpack_dht_data(void);
+
+uint8_t check;
```

Comment:
These should be prefaced with "g_" because they're global. Sorry, forgot to mention that before... 
While we're at it, some comments explaining what they're for wouldn't hurt.